### ISSUE TITLE
Typos in twisted.web._newclient.Request

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5591]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#5591|
|Type|defect|
|Created|2012-04-01 23:09:51Z|

  1. It says "connecton", instead of "connection", in the docstring for `twisted.web._newclient.Request`.
  1. It says "bofdy" instead of "body" in `Response._bodyDataReceived_FINISHED`.

Attachments:

* [firstPatch.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/295/29514305588a2a34d4ccd146b6cac2522175785f/5db70ccad990b012aca9019934052ea54b36bb99.diff) (971 bytes) - added by chhabraamit on 2012-04-02 19:55:19Z - 1. From twisted.web._newclient.Request. correcet connecton", to "connection" and in 'Response._bodyDataReceived_FINISHED.' , 'bodfy' to 'body'
* [firstPatch.2.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/295/29514305588a2a34d4ccd146b6cac2522175785f/884bdf020883df7e72c0a25db56a4735e347bf37.diff) (971 bytes) - added by chhabraamit on 2012-04-02 19:55:20Z - 1. From twisted.web._newclient.Request. correcet connecton", to "connection" and in 'Response._bodyDataReceived_FINISHED.' , 'bodfy' to 'body'

<details><summary>Searchable metadata</summary>

```
trac-id__5591 5591
type__defect defect
reporter__itamar itamar
priority__lowest lowest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__easy easy
time__1333321791000000 1333321791000000
changetime__1333407553000000 1333407553000000
version__None None
owner__exarkun exarkun
cc__jknight
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="chhabraamit's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|chhabraamit commented|
|-|-|

Replying to [DefaultCC Plugin](#note_1):

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun commented|
|-|-|

Thanks!  Looks like you accidentally added some text to the summary.  I'm adjusting that back to the original string.  Also, you can just use the comment box directly, you don't need to use the reply button (perhaps you discovered that already :).

One mechanical point about the patch.  Please generate it from the very top of the Twisted source tree, so that it has an "index" like "twisted/web/_newclient.py" instead of just "_newclient.py".  You can probably do this by just making sure your working directory is the top of your Twisted checkout, instead of `ThatPath/twisted/web` when you run `svn diff`. Patches constructed this way are easier to apply.

The change itself looks great.  My only suggestion is to include a [ReviewProcess#Newsfiles news fragment] for future changes.  This change only needs a `.misc` fragment, I think, which I think "svn diff" sometimes overlooks (because it is empty), so it's hard to include in a patch file anyway!  I can create it myself and apply your patch to trunk.

Thanks again!

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set status to `closed`|
|-|-|

(In [34060]) Fix typos in web client API documentation

Author: chhabraamit
Reviewer: exarkun
Fixes: [#5591](https://github.com/twisted/twisted/issues/5591)

Fix a couple simple misspellings in the API documentation for some `Agent`
support code.

**Comment by twisted-trac** [comment]
|<img alt="chhabraamit's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_7"></a>|chhabraamit commented|
|-|-|

Thank you for your comments . This was my first patch.
I have upload another same kind of patch for ticket 5593 .
Please let me know if I am missing something.
